src/views/QuestionnareModal/QuestionnareModal.tsx

```diff
@@ -50,6 +50,15 @@ const CssTextField = styled(TextField)({
   },
 })
 
+const pillarOrder: Record<string, number> = {
+  Identity: 1,
+  Devices: 2,
+  Networks: 3,
+  Applications: 4,
+  Data: 5,
+  CrossCutting: 6,
+}
+
```

Comment:
Remove this and just rely on the pillar order field. Dev will be updated soon with order values